config/psp.example.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
I think the format should be
```json
{
  "chains": [...],
  "triggers": {
    "proto-psp": [
      "SUBSCRIPTION ID #1 GOES HERE",
      "SUBSCRIPTION ID #2 GOES HERE",
      ...
    ]
  },
  "subscriptions": {
    "SUBSCRIPTION ID #1 GOES HERE": {
      "SUBSCRIPTION ID #1 FIELDS GO HERE"
    },
    "SUBSCRIPTION ID #2 GOES HERE": {
      "SUBSCRIPTION ID #2 FIELDS GO HERE"
    },
    ...
  },
  "templates": {
    "TEMPLATE ID #1 GOES HERE": {
      "TEMPLATE ID #1 FIELDS GO HERE"
    },
    "TEMPLATE ID #2 GOES HERE": {
      "TEMPLATE ID #2 FIELDS GO HERE"
    },
    ...
  }
}
```

Reply:
Here, proto-psp means the PSP implementation that doesn't care about allocators, authorizers, sponsorships. See the README comments for more info.

Reply:
Probably a better option to the above is to specify which subscription ID is supposed to be served by which slot. So it would go like
```json
{
  ...,
  "triggers": {
    "proto-psp": {
      "1": "SUBSCRIPTION ID #1 GOES HERE",
      "2": "SUBSCRIPTION ID #2 GOES HERE",
      ...
    }
  },
  ...
}
```
where `"1"` and `"2"` represent the slot indices. This would allow us to use allocators and hardcoded subscriptions simultaneously.
Btw I'm not sure if we should skip `0` as a slot index. We probably should.